roles/rundeck/defaults/main.yaml

```diff
@@ -2,14 +2,14 @@
 rundeck_enabled: yes
 rundeck_prune_dangling: yes
 
-rundeck_container_env_file: /etc/rdba/rundeck/server/config/env.cfg
-rundeck_container_realm_file: /etc/rdba/rundeck/server/config/realm.properties
-rundeck_container_extra_args: ' --ulimit=host -p 127.0.0.1:4440:4440 --volume /dev/null:/home/rundeck/etc/rundeckpro-license.key --volume /etc/rdba/rundeck/server/config/realm.properties:/home/rundeck/server/config/realm.properties --volume rundeck-data:/home/rundeck/server/data '
+rundeck_container_env_file: /usr/local/percona/rundeck/rundeck_env.cfg
+rundeck_container_realm_file: /usr/local/percona/rundeck/rundeck_realm.properties
+rundeck_container_extra_args: '--ulimit=host -p 127.0.0.1:4440:4440 --volume /dev/null:/home/rundeck/etc/rundeckpro-license.key --volume {{ rundeck_container_realm_file }}:/home/rundeck/server/config/realm.properties --volume {{ rundeck_container_volume }}:/home/rundeck/server/data'
 rundeck_container_image_repo: '{{ "docker.io/rundeck" }}'
 rundeck_container_port: 4440
 rundeck_container_volume: rundeck-data
 
-rundeck_deploy_using: podman
+rundeck_deploy_using: '{{ container_engine | mandatory }}'
```

Comment:
Doesn't this force `container_engine` to be defined in the inventory?

Reply:
Switched to first `container_engine_choices`

Reply:
How does that help? It doesn't

What is wrong with this?
`rundeck_deploy_using: '{{ container_engine | default("podman") }}'`